support-frontend/assets/pages/contributions-landing/contributionsLandingActions.js

```diff
@@ -476,6 +476,11 @@ const onPaymentResult = (paymentResult: Promise<PaymentResult>, paymentAuthorisa
               // Must re-render the wallet widget in order to display amazon's error message
               dispatch(setAmazonPayWalletWidgetReady(false));
             }
+            // Reset any updates the previous payment method had made to the form's billingCountry or billingState
+            dispatch(updateBillingCountry(''));
```

Comment:
I'm surprised flow allows `''`, shouldn't this be `null`?

Reply:
Billing country is of type: ?IsoCountry, which does not allow null, only a valid value or undefined.

Reply:
Where? In the reducer it's `billingCountry: IsoCountry | null,`
Flow should not allow empty string as a valid `IsoCountry`!

Reply:
Actually, I figured out a way to fix it.

Reply:
https://github.com/guardian/support-frontend/pull/2378/commits/641e0dfc9146f5dae5ad7527742d032ac821e401

Reply:
nice!